modules/nw-networkpolicy-audit-enable.adoc

```diff
@@ -0,0 +1,67 @@
+[id="nw-networkpolicy-audit-enable_{context}"]
+= Enabling network policy audit logging
+
+As a cluster administrator, you can enable network policy audit logging for a namespace.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. To enable network policy audit logging for a namespace, enter the following command:
++
+[source,terminal]
+----
+$ oc annotate namespace <namespace> \
+  k8s.ovn.org/acl-logging='{ "deny": "alert", "allow": "notice" }'
+----
++
+--
+where:
+
+`<namespace>`:: Specifies the name of the namespace.
+--
++
+[TIP]
+====
+You can alternatively apply the following YAML to enable audit logging:
+
+[source,yaml]
+----
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: <namespace>
+  annotations:
+    k8s.ovn.org/acl-logging: |-
+      {
+        "deny": "alert",
+        "allow": "notice"
+      }
+----
+====
++
+.Example output
+[source,terminal]
+----
+namespace/verify-audit-logging annotated
+----
+
+.Verification
+
+. Output the latest entries in the network policy audit log:
++
+[source,terminal]
+----
+$ for node in $(oc get pods -n openshift-ovn-kubernetes -l app=ovnkube-node --no-headers=true | awk '{ print $1 }') ; do
+    oc exec -it $node -n openshift-ovn-kubernetes -- tail -4 /var/log/ovn/acl-audit-log.log
+  done
+----
++
+.Example output
+[source,text]
```

Comment:
Here and elsewhere, while I am not offended by `source,text`, the team seems to have agreed (or conceded) to `source,terminal` for output too.

```
If you are also showing a code block for the output of the command, use [source,terminal] for that code block as well.
```

Reply:
@mikemckiernan, I must have missed that, or that movie _Catch me if you can_ comes to mind.

Reply:
@mikemckiernan, but I see someone stuffed that into the guidelines as well.